### PR TITLE
Create `WorkflowExecutionOptions` entity

### PR DESCRIPTION
**Pull request by isxoqovxasanboy**

### CODE REVIEW

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Common/Identity/Services/IAccountService.cs [hunk 1]
```diff
@@ -14,8 +15,11 @@ public interface IAccountService
     /// <param name="asNoTracking">Indicates whether to disable change tracking. Default is false.</param>
     /// <param name="cancellationToken">Cancellation token for asynchronous operations.</param>
     /// <returns>A <see cref="ValueTask"/> containing the retrieved user or null if not found.</returns>
-    ValueTask<Client?> GetUserByEmailAddressAsync(string emailAddress, bool asNoTracking = false,
-        CancellationToken cancellationToken = default);
+    ValueTask<Client?> GetUserByEmailAddressAsync(
+        string emailAddress,
+        QueryTrackingMode trackingMode = QueryTrackingMode.AsTracking,
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Common/Identity/Services/IAccountService.cs:20]: Pass `QueryOptions` instead

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/WorkflowExecutionOptionRepository.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+using Feedback.Analyzer.Domain.Entities;
+using Feedback.Analyzer.Persistence.Caching.Brokers;
+using Feedback.Analyzer.Persistence.DataContexts;
+using Feedback.Analyzer.Persistence.Extensions;
+using Feedback.Analyzer.Persistence.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace Feedback.Analyzer.Persistence.Repositories;
+
+public class WorkflowExecutionOptionRepository(AppDbContext dbContext, ICacheBroker cacheBroker)
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/WorkflowExecutionOptionRepository.cs:11]: Add type docs

**Review by sulton-max — CHANGES_REQUESTED** [review]
Good job man, just recreate the migrations that's it 👍

**Review by sulton-max — DISMISSED** [review]
Well done 👏👍

**Review by rashidovasadbek — APPROVED** [review]

**Review by sulton-max — APPROVED** [review]

**Review by Ilxom-22 — APPROVED** [review]

**Pull request merged**